app/src/main/java/de/westnordost/streetcomplete/data/QuestType.kt

```diff
@@ -10,6 +10,9 @@ interface QuestType<T> {
     /** the title resource id used to display the quest's question */
     val title: Int
 
+    /** returns on or more replacements, to be used in the title template. The tags are unmodifiable. */
```

Comment:
> The tags are unmodifiable

By the way, as opposed to Java, in Kotlin, the collection interfaces `Map`, `List`, etc. are all immutable. The mutable counterparts are `MutableMap`, `MutableList`, etc.

Reply:
Also, `getTitleReplacements`should go into `OsmElementQuestType` as well, of course